carbonmark/components/Transaction/Approve.tsx

```diff
@@ -67,23 +68,6 @@ export const Approve: FC<Props> = (props) => {
           }
           iconName={props.amount.token}
         />
-        {!!props.price && (
```

Comment:
How come that the price is being removed here?
Can you point to the figma designs for that?

This Transaction Modal is used here:
- Create Listing
- Edit Listing
- Purchase a Listing

Editing a Listing always comes with the price.

Reply:
| Before WITH PRICE | After WITHOUT PRICE  |
|---------|--------|
| <img width="400" alt="Bildschirmfoto 2023-05-19 um 14 37 53" src="https://github.com/KlimaDAO/klimadao/assets/95881624/35fa1eeb-c4c7-4179-a4d1-9e15482b8e7f">  | <img width="400" alt="Bildschirmfoto 2023-05-19 um 14 37 53" src="https://github.com/KlimaDAO/klimadao/assets/95881624/9fc65859-d8db-4766-ab17-7a9b9042e7a3">
 |

Reply:
> Hey @psparacino
> 
> Having a quick glance at this PR as I am working on retirement from Project details (and may use your code here and there)
> 
> See some comments below.
> 
> Also I was not able to continue with the retirement on the preview link. You may be aware of that?
> 
> https://carbonmark-git-tufnel-retire-from-cm-plumbing-1-7961eb-klimadao.vercel.app/portfolio/0xd6ed6fae5b6535cae8d92f40f5ff653db807a4ea/retire <img alt="Bildschirmfoto 2023-05-19 um 14 49 55" width="1445" src="https://user-images.githubusercontent.com/95881624/239536634-4150055a-8a6e-41bd-92fa-0333db535b1d.png">

Thanks for taking a look; addressing comments individually below.

Re: retirement on the preview link, might be two reasons:

1) I was testing with a TC02 token; looks like you're using a C3T so I'll go back and confirm there are icons for all the carbon tokens (which looks like the reason for this failure).

2) When I tried using a TC02 token on the preview I still got a failure with `token` being undefined so maybe it has to do something with the env(?) in vercel? During development I had to specify the export `const DEFAULT_NETWORK = "mainnet";` otherwise it would default to testnet

Reply:
Yes, price should not be removed on create/edit, so fixing that. it does look the design change is across the board though for all transactions; this is the create listing design in figma:
<img width="274" alt="create design" src="https://github.com/KlimaDAO/klimadao/assets/72105234/67f74b74-e1da-48b6-96e2-8bb90b1a8f3f">

Create Approve:
<img width="160" alt="create approve design" src="https://github.com/KlimaDAO/klimadao/assets/72105234/ab4958be-e783-4109-95c2-767584c05c6c">